src/entt/signal/dispatcher.hpp

```diff
@@ -248,6 +253,31 @@ class dispatcher {
         }
     }
 
+    /**
+     * @brief Returns the count of pending events for the specified type.
+     *
```

Comment:
```suggestion
     * @tparam Event Type of event for which to return the count.
     * @return The count of pending events for the specified type.
```
Please, remove the comment below too.